src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/gadgets/MultiTool.java

```diff
@@ -67,8 +69,11 @@ protected ItemUseHandler getItemUseHandler() {
             int index = selectedMode.getOrDefault(p.getUniqueId(), 0);
 
             if (!p.isSneaking()) {
+                SlimefunItem sfItem = modes.get(index).getItem();
+                if (sfItem.getID().equals(SlimefunItems.STAFF_WIND.getItemId()) && (SlimefunUtils.isItemSimilar(p.getInventory().getItemInMainHand(), SlimefunItems.STAFF_WIND, false, false) || SlimefunUtils.isItemSimilar(p.getInventory().getItemInOffHand(), SlimefunItems.STAFF_WIND, false, false))) {
+                    return;
+                }
```

Comment:
Also: Why did you add this? It doesn't make any sense to add that here

Reply:
> but when testing I found that this causes the player to loose 4 hunger instead of 2 because the multi tool is no longer caught.